server/index.js

```diff
@@ -138,40 +136,43 @@ app.get(
   '/auth/google/callback',
   passport.authenticate('google', { failureRedirect: '/auth/google' }),
   function (req, res) {
-    res.redirect('http://localhost:3000/'); // 포트 번호 변경 필요
+    res.redirect(`http://${HOST}:${PORT}/auth`);
   },
 );
 
-app.get('/', (req, res) => {
+
+app.get('/auth', (req, res) => {
   if (req.isAuthenticated()) return res.status(200).send(req.user);
   res.status(401).send({ msg: 'logout' });
 });
 
-app.get('/signout', (req, res) => {
+
+app.get('/auth/signout', (req, res) => {
   console.log('현재 사용자를 로그아웃 합니다.');
   req.logout();
-  res.redirect('/');
+  res.redirect('/auth');
 });
 
+
 app.post(
-  '/signin',
+  '/auth/signin', 
   passport.authenticate('local', {
-    failureRedirect: '/',
+    failureFlash : true
```

Comment:
failureFlash 몰라서 찾아봤는데 인증이 실패한 이유를 메세지 창으로 띄워주는 거지? 근데 failureRedirect 는 실패 시 돌아갈 경로니까 같이 입력하는 편이 좋지 않을까?
```suggestion
    failureRedirect: '/auth',
    failureFlash : true
```

Reply:
아아 자세하게 설명을 안적었네.. 번거롭게 해서 죄송합니다🥲
 `failureRedirect` 지정하면 무조건 `msg : logout` 만 띄워서 차선책으로 저렇게 했어유! 

자세한 내용 api-document에 같이 적어서 서버에 띄워보겠습니다 !

Reply:
아하 failureRedirect가 제대로 작동이 안돼서 그랬군요 감사합니다😆